### PR TITLE
Fixed endDate bugs caused by UserProfile refactoring and moment forma…

### PR DESCRIPTION
**Pull request by calvertwong**
Date wasn't converted to the correct format and it is now fixed with the right formatting.

Fix wrongly converted date in user management

Display end date correctly under the volunteer name
Show end date under volunteer name

Display N/A when end date does not exist
NA when end date does not exist

### CODE REVIEW

File: src/components/UserManagement/UserTableData.jsx [hunk 1]
```diff
@@ -60,7 +61,7 @@ const UserTableData = React.memo((props) => {
           ? (props.user.reactivationDate.toLocaleString().split('T')[0]) : ''}
       </td>
       <td>
-        {props.user.endDate ? (props.user.endDate.toLocaleString().split('T')[0]) : 'N/A'}
+        {props.user.endDate ? (moment(props.user.endDate).format('YYYY-MM-DD')) : 'N/A'}
```
Comment (calvertwong) [inline src/components/UserManagement/UserTableData.jsx:64]: This is what caused the date to show as a long number as shown in the link below
https://user-images.githubusercontent.com/18335122/132959248-5d9ef39a-7751-4a48-b56f-810307f93462.jpg

File: src/components/UserProfile/UserProfile.jsx [hunk 2]
```diff
@@ -432,7 +432,7 @@ const UserProfile = props => {
               <p className="proile-rating">
                 From : <span>{moment(userProfile.createdDate).format('YYYY-MM-DD')}</span>
                 {'   '}
-                To: <span>N/A</span>
+                To: <span>{moment(userProfile.endDate).format('YYYY-MM-DD')}</span>
```
Comment (calvertwong) [inline src/components/UserProfile/UserProfile.jsx:435]: Display end date properly under the volunteer name

File: src/components/UserProfile/UserProfile.jsx [hunk 3]
```diff
@@ -432,7 +432,7 @@ const UserProfile = props => {
               <p className="proile-rating">
                 From : <span>{moment(userProfile.createdDate).format('YYYY-MM-DD')}</span>
                 {'   '}
-                To: <span>N/A</span>
+                To: <span>{userProfile.endDate ? moment(userProfile.endDate).format('YYYY-MM-DD') : 'N/A'}</span>
```
Comment (calvertwong) [inline src/components/UserProfile/UserProfile.jsx:435]: Display end date properly under the volunteer name

**Review by chrisweilacker — COMMENTED** [review]
lgtm can move this in

**Comment by calvertwong** [comment]
Could you please approve it? The merge button is disabled when there is no approval.

**Review by chrisweilacker — APPROVED** [review]
lgtm

**Pull request merged**